cuda_core/cuda/core/experimental/_stream.py

```diff
@@ -235,6 +319,15 @@ def __init__(self):
 
 
 def default_stream():
+    """Return the default CUDA :obj:`Stream`.
```

Comment:
Note: This function is internal so we don't have to document it. We can still keep it as-is, though.

Reply:
Noted